packages/editor/src/components/projectEditor/editors/contractConfigModal/ContractConfigModal.tsx

```diff
@@ -164,6 +175,29 @@ export default class ContractConfigModal extends Component<IProps, IState> {
                                         onChange={this.onNameChange}
                                     />
                                 </div>
+                                <div className={style.valueContainer}>
+                                    <div className='superInputDark'>
+                                        <label htmlFor='value'>
+                                            Value
+                                            <Tooltip html={
+                                                    <div className='arrayInfoTooltip'>
+                                                        <div>Value is in wei units</div>
+                                                        <div className='example'>this field can be used in case if contract has payable constructor and needs initial funding</div>
+                                                        <div className='example'>`for example constructor() public payable {'{ ... }'}</div>
```

Comment:
Consider better wording and formatting:
```
<div>Value is in Wei units</div>
<div className='example'>This field can be used when a contract defines payable constructor and requires initial funding</div>
<div className='example'>For example: constructor() public payable {'{ ... }'}</div>
```

Reply:
Updated with the recommended wording. Thank you.